widget/button.go

```diff
@@ -661,6 +661,49 @@ func (b *Button) Click() {
 	}
 }
 
+// Press presses the button emulating a Mouse Left click
+func (b *Button) Press() {
+	b.init.Do()
+
+	offx := b.widget.Rect.Dx()
+	offy := b.widget.Rect.Dy()
+
+	// This means that there are some pixels that are not clickable
+	if b.mask != nil {
+		offx = offx / 2
+		offy = offy / 2
+	}
+	b.hovering = true
+	b.widget.MouseButtonPressedEvent.Fire(&WidgetMouseButtonPressedEventArgs{
+		Widget:  b.widget,
+		Button:  ebiten.MouseButtonLeft,
+		OffsetX: offx,
+		OffsetY: offy,
+	})
+}
+
+// Release releases the button emulating a Mouse Left release
+func (b *Button) Release() {
+	b.init.Do()
+
+	offx := b.widget.Rect.Dx()
+	offy := b.widget.Rect.Dy()
+
+	// This means that there are some pixels that are not clickable
+	if b.mask != nil {
+		offx = offx / 2
```

Comment:
assignOp: replace `offx = offx / 2` with `offx /= 2` (gocritic)
for all 4 spots